src/sources/ljm/main.py

```
@@ -198,6 +211,20 @@ def main():
         num_addresses, a_scan_list_names = calibration.Sensor.setup(handle)
         a_scan_list = ljm.namesToAddresses(num_addresses, a_scan_list_names)[0]
 
+        # Stream Configuration
+        stream_info.aScanListNames = aScanListNames
```

Ruff (0.15.5)

[error] Undefined name `stream_info`

(F821)